### DISCUSSION TITLE
Uninstall SDM

### DISCUSSION THREAD

**Discussion by tiefenauer** [gitbls/sdm#49] [Q&A] [answered]
I installed SDM using curl to download the [EZsdmInstaller](https://github.com/gitbls/sdm/blob/master/EZsdmInstaller) script and piping to bash as described in the installation instructions. Is there an easy way to remove SDM?

**Comment by gitbls** [comment, marked as answer]
```
sudo rm -rf /usr/local/sdm
sudo rm -f /usr/local/bin/sdm
```

**Reply by tiefenauer** [reply]
added https://github.com/gitbls/sdm/pull/52 for this
